scripts/create_full_recipients_setup.py

```diff
@@ -15,18 +15,50 @@
 
 from brownie import AllowedRecipientsBuilder
 
+'''
+Please fill out deploy_config before running the script.
+
+A. If you want a new token registry to be created when the script is executed:
+- fill in the "tokens" parameter in the deploy_config with a list of tokens to be added to the registry
+- leave the "tokens_registry" parameter empty
+
+    Example:
+    tokens=["0x2EB8E9198e647f80CCF62a5E291BCD4a5a3cA68c", "0x86F6c353A0965eB069cD7f4f91C1aFEf8C725551"],
+    tokens_registry = "",
+
+B. If you prefer to use an existing token registry when the script is executed:
+- fill in the "tokens_registry" parameter in the deploy_config with the address of the token registry that should be used
+- fill in the "tokens" parameter with a list of tokens that are included in the specified registry - this will be used later during testing 
+
+    Example:
+    tokens=["0x2EB8E9198e647f80CCF62a5E291BCD4a5a3cA68c", "0x86F6c353A0965eB069cD7f4f91C1aFEf8C725551", "0x9715b2786F1053294FC8952dF923b95caB9Aac42"],
+    tokens_registry = "0x091c0ec8b4d54a9fcb36269b5d5e5af43309e666",
+
+The "tokens_registry" parameter of the deploy_config is used primarily to verify the method of contracts deployment. 
+Please make sure you have filled deploy_config correctly!
+'''
+
 deploy_config = deployment.AllowedRecipientsFullSetupDeployConfig(
     tokens=[],
+    tokens_registry = "",
     limit=0, 
     period=1,
     spent_amount=0,
-    trusted_caller="",  
-    titles=[""],
-    recipients=[],
+    titles=["", ""],
+    recipients=["", ""],
+    trusted_caller="",
+    grant_rights = False, # permissions to execute AddAllowedRecipient / RemoveAllowedRecipient methods on behalf of trusted_caller
 )
 
 
 def main():
+
+    new_token_registry_is_required = not bool(deploy_config.tokens_registry)
+
+    if new_token_registry_is_required and not bool(deploy_config.tokens):
```

Comment:
I would add a check that both fields tokens and tokens_registry are filled to avoid misbehaviour